src/AwesomeAlert.js

```diff
@@ -166,31 +170,31 @@ export default class AwesomeAlert extends Component {
     };
 
     return (
-      <View style={[styles.container, alertContainerStyle]}>
-        <TouchableWithoutFeedback onPress={this._onTapOutside}>
-          <View style={[styles.overlay, overlayStyle]} />
-        </TouchableWithoutFeedback>
-        <Animated.View
-          style={[styles.contentContainer, animation, contentContainerStyle]}
-        >
-          <View style={[styles.content, contentStyle]}>
-            {showProgress ? (
-              <ActivityIndicator size={progressSize} color={progressColor} />
-            ) : null}
-            {title ? (
-              <Text style={[styles.title, titleStyle]}>{title}</Text>
-            ) : null}
-            {message ? (
-              <Text style={[styles.message, messageStyle]}>{message}</Text>
-            ) : null}
-            {customView}
-          </View>
-          <View style={[styles.action, actionContainerStyle]}>
-            {showCancelButton ? this._renderButton(cancelButtonData) : null}
-            {showConfirmButton ? this._renderButton(confirmButtonData) : null}
-          </View>
-        </Animated.View>
-      </View>
+        <View style={[styles.container, alertContainerStyle]}>
```

Comment:
same as above, the green square you're seeing is the extra unwanted spacing that we want to remove. This change should not show up in the PR diff.